src/assets/styles/modules/blog-post.scss

```diff
@@ -0,0 +1,15 @@
+/* ================ Blog post ================ */
+.blog-post {
+  display: grid;
+  grid-column-gap: 15px;
+  grid-template-columns: 1fr 1fr 1fr;
+}
+
+.blog-post__date {
+  display: block;
+  margin-bottom: 7px;
+}
+
+.blog-post__excerpt {
```

Comment:
Don't think we need this style or class. This style is subjective and probably will change when you style your theme.

Reply:
I removed it.